packages/eas-cli/tsconfig.json

```diff
@@ -10,7 +10,7 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "strict": true,
-    "target": "ES2021",
```

Comment:
we should probably extend one of the tsconfig bases (https://github.com/tsconfig/bases#centralized-recommendations-for-tsconfig-bases) to ensure this target is compatible with the minumum node version that we support for eas-cli.

That being said, I checked and we support node >=18 which uses es2022 in the base: https://www.npmjs.com/package/@tsconfig/node18

Reply:
https://github.com/expo/eas-cli/pull/2739